root/programs/C#/Frameworks/Infrastructure/Business/Util/MyCmnFunction.cs

```diff
@@ -28,56 +28,73 @@
 //*  日時        更新者            内容
 //*  ----------  ----------------  -------------------------------------------------
 //*  20xx/xx/xx  ＸＸ ＸＸ         新規作成（テンプレート）
-//*  2009/06/02  西野  大介        sln - IR版からの修正
+//*  2009/06/02  西野 大介         sln - IR版からの修正
 //*                                ・#5  ： コントロール数取得処理（デフォルト値不正）
-//*  2009/07/21  西野  大介        コントロール取得処理の仕様変更
-//*  2009/08/10  西野  大介        同名のコントロール追加に対応（GridView/ItemTemplate）。
-//*  2010/09/24  西野  大介        ジェネリック対応（Dictionary、List、Queue、Stack<T>）
+//*  2009/07/21  西野 大介         コントロール取得処理の仕様変更
+//*  2009/08/10  西野 大介         同名のコントロール追加に対応（GridView/ItemTemplate）。
+//*  2010/09/24  西野 大介         ジェネリック対応（Dictionary、List、Queue、Stack<T>）
 //*                                nullチェック方法、Contains → ContainsKeyなどに注意
-//*  2010/10/21  西野  大介        幾つかのイベント処理の正式対応（ベースクラス２→１へ）
-//*  2012/06/14  西野  大介        コントロール検索の再帰処理性能の集約＆効率化。
-//*  2014/05/16  西野  大介        キャスト可否チェックのロジックを見直した。
+//*  2010/10/21  西野 大介         幾つかのイベント処理の正式対応（ベースクラス２→１へ）
+//*  2012/06/14  西野 大介         コントロール検索の再帰処理性能の集約＆効率化。
+//*  2014/05/16  西野 大介         キャスト可否チェックのロジックを見直した。
+//*  2017/01/31  西野 大介         System.Webを使用しているCalculateSessionSizeメソッドをPublicから移動
 //**********************************************************************************
 
-using System.Text;
-
-// System
 using System;
-using System.Collections;
 using System.Collections.Generic;
 
-// System.Web
 using System.Web;
-using System.Web.Security;
-
 using System.Web.UI;
 using System.Web.UI.WebControls;
-using System.Web.UI.WebControls.WebParts;
-using System.Web.UI.HtmlControls;
-
-// 業務フレームワーク（循環参照になるため、参照しない）
 
-// フレームワーク
-using Touryo.Infrastructure.Framework.Business;
-using Touryo.Infrastructure.Framework.Common;
-using Touryo.Infrastructure.Framework.Dao;
-using Touryo.Infrastructure.Framework.Exceptions;
-using Touryo.Infrastructure.Framework.Presentation;
 using Touryo.Infrastructure.Framework.Util;
-using Touryo.Infrastructure.Framework.Transmission;
-
-// 部品
-using Touryo.Infrastructure.Public.Db;
 using Touryo.Infrastructure.Public.IO;
-using Touryo.Infrastructure.Public.Log;
-using Touryo.Infrastructure.Public.Str;
 using Touryo.Infrastructure.Public.Util;
 
 namespace Touryo.Infrastructure.Business.Util
 {
     /// <summary>Business層の共通クラス</summary>
     public class MyCmnFunction
     {
+        #region CalculateSessionSize
+
+        /// <summary>Sessionサイズ測定</summary>
+        /// <returns>Sessionサイズ（MB）</returns>
+        /// <remarks>シリアル化できないオブジェクトを含む場合は落ちる。</remarks>
+        public static long CalculateSessionSizeMB()
+        {
+            return MyCmnFunction.CalculateSessionSizeKB() / 1000;
```

Comment:
@daisukenishino 

I think that the result value should be rounded off.
For example, if *MyCmnFunction.CalculateSessionSizeKB* method returns *999*, this method returns *0*.